R/define.R

```diff
@@ -18,6 +18,10 @@
 
 #' Define enrollment rate
 #'
+#' The function define subject enrollment rate for a study.
```

Comment:
1. define -> define**s**
2. Avoid starting the sentence with obvious stuff like "This function..." or "This package...".